Assets/Scripts/UI/Menu's/MenuController.cs

```diff
@@ -30,11 +34,18 @@ public void ToggleToPauseScreenFromOptions(bool goingToOptions)
 
     private void Start() => activePauseScreen.SetActive(false);
 
-    private void Update() => UpdatePauseMenu();
+    private void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.I))
+        {
+            LosingCondition();
+        }
+        UpdatePauseMenu();
+    }
 
     private void UpdatePauseMenu()
     {
-        if (!Input.GetKeyDown(KeyCode.Tab)) return;
+        if (!Input.GetKeyDown(KeyCode.Tab) || _hasLost) return;
```

Comment:
```suggestion
        if (_hasLost || !Input.GetKeyDown(KeyCode.Tab)) return;
```
Maakt niet veel uit, maar de bool is in dit script en is daardoor sneller te checken.